From 328bd27afd2989d52f000ade9b07b4d00479c59a Mon Sep 17 00:00:00 2001
From: AsavarTzeth <asavartzeth@gmail.com>
Date: Sat, 16 Feb 2019 16:36:29 +0100
Subject: [PATCH 1/3] Fix building & installing only lv2

This solves building and installing the lv2 components, without the need
to build everything else. It will still complain about the lack of jack,
but that is not a problem since the lv2 plugins does not need it.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c918fb11..2e18be0a 100644
--- a/Makefile
+++ b/Makefile
@@ -174,7 +174,7 @@ install_ladspa: all
 	@mkdir -p $(DESTDIR)$(LADSPA_PATH)
 	@$(INSTALL) $(LIB_LADSPA) $(DESTDIR)$(LADSPA_PATH)/
 	
-install_lv2: all
+install_lv2: build_lv2
 	@echo "Installing LV2 plugins to $(DESTDIR)$(LV2_PATH)/$(ARTIFACT_ID).lv2"
 	@mkdir -p $(DESTDIR)$(LV2_PATH)/$(ARTIFACT_ID).lv2
 	@$(INSTALL) $(LIB_LV2) $(DESTDIR)$(LV2_PATH)/$(ARTIFACT_ID).lv2/
-- 
2.20.1

